docs/UserGuide.md

```diff
@@ -181,39 +214,51 @@ Example:
 
 ---
 
-### Exiting the program : `exit`
+### Undoing a command : `undo`
 
-Exits the program.
+Undoes the most recently used command. No changes if no previous state exists.
 
-Format: `exit`
+Format: `undo`
 
 ---
 
-#### Manually saving data
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note for manually saving data:**<br>
 
 Animal data is saved in the hard disk with the preferred file name automatically after any command that changes the data. 
 By default, the preferred file name is `zookeepbook.json`
-There is no need to save manually, though a copy of the current data can be created with the `snap` command. 
+There is no need to save manually, though you can create a copy of the current data with the `snap` command which will be explained below.
+
+</div>
 
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous ZooKeep home folder.
 
+**Q**: Can I record the feeding times of a specific animal in any order I prefer?
+**A**: The feeding time will be arranged in chronological order regardless of the order entered for easier reference.
+
+**Q**: Can I search for animals based on a certain alphabet or half specified keywords?
+**A**: The find feature will only list animals with the exact specified keyword provided.
```

Comment:
maybe can include that it is case insensitive?

Reply:
I think i placed it as a tip under the find feature, thanks anyways!